### PR TITLE
Fix Kuzzle version in Docker files

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
## What does this PR do ?
Fix Kuzzle version in Docker files (docker-compose & Dockerfiles) 

### How should this be manually tested?
Try to run the docker-compose.yml stack

### Other changes
Make the Readme looks great

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,40 +26,6 @@ jobs:
         - npm run codecov
         - bash test/travis-bin/sonar.sh
 
-    # -------------------------------------------------------------------------
```
Comment (Aschen) [inline .travis.yml]: Unit tests should use Node.js 10 (top of `.travis.yml` file)

File: README.md [hunk 2]
```diff
@@ -15,15 +34,19 @@ It also provides you with a flexible and powerful user-management system.
...
 Train yourself and your teams to use Kuzzle to maximize its potential and accelerate the development of your projects.  
 Our teams will be able to meet your needs in terms of expertise and multi-technology support for IoT, mobile/web, backend/frontend, devops.  
 :point_right: [Get a quote](https://hubs.ly/H0jkfJ_0)
 
-## Compatibility
+### Compatibility matrice
+
+| Kuzzle Version | Plugin Version |
+| -------------- | -------------- |
+| 1.x.x          | 3.x.x          |
```
Comment (Aschen) [inline README.md]:
```suggestion
| 1.8.x          | 3.x.x          | 
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
 
-# Cluster mode plugin
 
-This plugin adds a cluster mode to Kuzzle.
+## About
 
-## Kuzzle
+### Kuzzle Cluster Plugin
+
+This plugin adds a masterless cluster mode to Kuzzle.
```
Comment (Aschen) [inline README.md]: You can add a link to the scalability guide here (like in the other PR)

Comment (alexandrebouthinon) [reply README.md]: Done :slightly_smiling_face:

**Comment by alexandrebouthinon** [comment]
I made the changes and the release tool have been updated :slightly_smiling_face:

File: docker-compose/config/pm2-dev.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "apps": [{
     "name"         : "kuzzle",
-    "script"       : "bin/kuzzle",
+    "script"       : "bin/start-kuzzle-server",
     "cwd"          : "/var/app",
     "args"         : ["start"],
```
Comment (Aschen) [inline docker-compose/config/pm2-dev.json]:
```suggestion
    "args"         : [],
```

**Pull request merged**